skipscale/config.py

```diff
@@ -17,6 +17,7 @@
 
 tenant_overrideable_fields = {
     schema.Optional('default_quality'): int, # default 85
+    schema.Optional('default_format'): str,
```

Comment:
`planner.py` has this schema for the format field:

```
    Optional('format'): And(str, Use(str.lower), lambda s: s in ('jpeg', 'png', 'webp')),
```

Perhaps have that here, too?